get-error-lines.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Generally a good idea to run [shellcheck](https://www.shellcheck.net/) on any bash script and fix the problems it identifies, because bash scripting is fraught with footguns.